src/WebSocketApi.h

```diff
@@ -23,9 +23,7 @@ with this program. If not, see <https://www.gnu.org/licenses/>
 #include <map>
 #include <mutex>
 #include <shared_mutex>
-#include <obs.h>
-
-#include "../lib/obs-websocket-api.h"
+#include <obs-websocket-api.h>
```

Comment:
Probably semantics, but removing the `obs.h` include here makes sense if the API implementation here does not use `libobs` API methods by itself.

The moment it directly does so, it needs to take care of the include itself and cannot rely that the websocket-API itself implicitly includes everything it needs.

Reply:
In practice the pre-processor might resolve this automatically, but that makes it harder for maintenance:

The `#include` directives at the top of a header also serve as documentation (as in: "These are the headers/libraries I implement functionality from").